services/src/northbound/src/main/java/org/openkilda/northbound/service/BasicService.java

```diff
@@ -81,4 +87,24 @@ default InfoData validateInfoMessage(final Message requestMessage,
         }
         return data;
     }
+
+    /**
+     * Validates switch existing.
+     *
+     * @param correlationId the correlation id.
+     * @param data list with source and destination switches.
+     */
+    default void validateSwitches(String correlationId, List<SwitchInfoData> data, FlowPayload flowPayload) {
```

Comment:
Please omit default.

Reply:
No. Keyword "default" must be here if method implemented in interface.